falcon-taz/src/main/java/pt/haslab/taz/TraceProcessor.java

```diff
@@ -537,27 +537,27 @@ private void parseMessageHandlers()
 
             // Use a fast and a slow iterator to handle nested message handlers while iterating through the thread events.
             SortedSet<Event> threadEvents = eventsPerThread.get( thread );
-            Iterator<Event> slowIt = threadEvents.iterator();
-            Iterator<Event> fastIt = threadEvents.iterator();
+            Iterator<Event> it = threadEvents.iterator();
```

Comment:
Can we rename this to make it developer-friendly? :D

Reply:
Sure :+1: Is `threadIterator` ok?

Reply:
Fixed in 6903f5a53dc1a317b2dc8b638cb1036a89eba817